source/fundamentals/typescript.txt

```diff
@@ -123,36 +123,46 @@ The type error raised by the preceding code snippet is as follows:
 Dot Notation in Find Operations
 ```````````````````````````````
 
-Starting in version 5.0, the {+driver-short+} does not provide type
-safety for *find operations* that search on fields expressed in dot
-notation. When you construct a filter to pass to a find method, such as
-``find()`` or ``findOneAndUpdate()``, your code compiles even if you
-specify a value of an incorrect type for a field expressed in dot
-notation.
+Starting in version 5.0, by default, the {+driver-short+} does not provide type
+safety for find operations that search on fields expressed in dot
+notation. When you construct a filter to pass to a find method, the
+driver will not raise a type error even if you specify an incorrectly
+typed value for a field expressed in dot notation.
 
-The following code compiles, even though the value of
-``classification.color`` should be a string:
+.. note::
+
+   The following methods comprise find operations:
```

Comment:
I: [Style Guide recommends not using "comprise" at all](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#c) (even using it correctly, which is weird)

S:
```suggestion
   The following methods are considered find operations:
```
Or:
```suggestion
   The following methods are find operations:
```
Or something similar.